web/src/features/projects/server/projectsRouter.ts

```diff
@@ -139,6 +99,7 @@ export const projectsRouter = createTRPCRouter({
       await ctx.prisma.project.delete({
         where: {
           id: input.projectId,
+          orgId: ctx.session.orgId,
```

Comment:
Should we also have an orgId check in line 65?

Reply:
yes, does not hurt, currently projectId is globally unique though